apps/web/src/design-system/switch/Switch.styles.ts

```diff
@@ -0,0 +1,26 @@
+import { createStyles, MantineTheme } from '@mantine/core';
+
+export default createStyles((theme: MantineTheme, _params, getRef) => {
+  const dark = theme.colorScheme === 'dark';
+  const label = getRef('label');
+  return {
+    input: {
+      backgroundColor: dark ? theme.colors.dark[5] : theme.colors.gray[5],
+      borderColor: 'transparent',
+      '&:checked': {
+        backgroundImage: theme.colors.gradient[8],
+        [`& + .${label}`]: {
+          backgroundImage: theme.colors.gradient[8],
+          backgroundClip: 'text',
+          color: 'transparent',
+        },
+      },
+    },
+    label: {
```

Comment:
Seems like the round ineer circle should be 20x20 but in the PR it's 18x18
![CleanShot 2022-02-06 at 19 16 25](https://user-images.githubusercontent.com/8872447/152692575-45572ed5-7352-4d36-ab1a-4027764ff3f2.png)